src/abstractdataframe/abstractdataframe.jl

```diff
@@ -2,41 +2,14 @@
     AbstractDataFrame
 
 An abstract type for which all concrete types expose an interface
-for working with tabular data.
+for working with tabular data. In DataFrames.jl the `DataFrame` and
+`SubDataFrame` types are subtypes of `AbstractDataFrame`.
```

Comment:
I'd put this after the sentence below which gives the overall definition.

Reply:
done